### DISCUSSION TITLE
Have you contributed to Paketo?

### DISCUSSION THREAD

**Discussion by davidmirror-ops** [paketo-buildpacks/feedback#35] [Show and tell]
It's your time to shine!

Comment on this Discussion telling the community about a contribution you have made to Paketo Buildpacks

Anything can count including:

1. A piece of content (a blog post, presentation, talk, live stream, etc)
2. A piece of code (merged Pull Request)
3. An interaction with the community: questions you have posted/answered on Slack/mailing list, issues you have created, proposals you have (co)authored

From all the Paketo community, **thank you** for your commitment to the project!

**Comment by loewenstein** [comment]
Thanks @davidmirror-ops for this opportunity and forum. I've asked my team to collect our contributions, which are [spread wide in the Paketo org](https://github.com/search?p=1&q=is%3Apr+author%3Apbusko+author%3Aphil9909+author%3Ac0d1ngm0nk3y+author%3Akulhadia+author%3Amodulo11+org%3Apaketo-buildpacks&type=Issues).

We've done small things like adding floating semantic version tags from [RFC](https://github.com/paketo-buildpacks/rfcs/pull/107) to implementation in [github-config](https://github.com/paketo-buildpacks/github-config/pull/378) and [pipeline-builder](https://github.com/paketo-buildpacks/pipeline-builder/pull/468) or passing JVM versions from Maven to the JVM distribution buildpacks.
We've added helpers to occam, fixed typos in READMEs, jumped on the Jammy stacks and provided fixes [npm-install](https://github.com/paketo-buildpacks/npm-install/pull/379), [npm-start](https://github.com/paketo-buildpacks/npm-start/pull/224), [apache-tomcat](https://github.com/paketo-buildpacks/apache-tomcat/pull/248) and [jam](https://github.com/paketo-buildpacks/jam/pull/107).

I guess we have been touching tooling, stacks, java and node but none of this too deeply. I'd be curious to hear how this would be reflected in the [roles](https://github.com/paketo-buildpacks/community/blob/main/GOVERNANCE.md#roles), as we won't easily fit into a sub team and besides, contribution will be a bit difficult to assign, as all are done in pairs.

> From all the Paketo community, **thank you** for your commitment to the project!

All thanks go directly to @pbusko, @phil9909, @c0d1ngm0nk3y, @kulhadia and @modulo11.

**Reply by phil9909** [reply]
@davidmirror-ops We are planning on doing further contributions and in the long run become "Contributor" or even "Maintainer" in some sub-teams.

Anything against promoting me and my colleagues to "Member" as a first step?

**Reply by davidmirror-ops** [reply]
@phil9909 Thank you so much for your contributions so far and your interest in doing even more for the project, we really appreciate it.

According to GOVERNANCE the only requirement for the Member role is to sign the CLA (which I'm sure you all already did) so nothing in that regard against the promotion. I will let @ryanmoran @dmikusa chime in, mainly to confirm how this has been done in the past in terms of new org member's permissions and things like that.

Thank you again!

**Reply by davidmirror-ops** [reply]
In the meantime, we were wondering if you'd like to see SAP in the [ADOPTERs](https://github.com/paketo-buildpacks/community/blob/main/ADOPTERS.md) list?
